Database.md

```diff
@@ -187,7 +196,7 @@ WHERE
 #### List directory
 
 ```sql
-SELECT uuid, name, archive_uuid 
+SELECT uuid, name, archive_uuid, size, is_shared 
```

Comment:
Update list query as the `is_shared` column has been removed

Reply:
As I say in the other PR (https://github.com/hawks-atlanta/docs/pull/23) lets continue development and update documentation as needed